### PR TITLE
[ADF-3274] people widget instead of the people list

### PR DESCRIPTION
**Pull request by bbcodrin**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [X] Tests for the changes have been added (for bug fixes / features)
> - [X] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [X] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-3274
Right now, the start form component is using the people list to render the assignee. 

**What is the new behaviour?**
We should use the people widget to achieve that as we do for the adf form.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services/task-list/components/start-task.component.html [hunk 1]
```diff
@@ -65,7 +65,7 @@
                 </div>
 
                 <div class="adf-grid-column adf-grid-half-width">
-                    <adf-people-selector [(peopleId)]="assigneeId" id="assignee_id"></adf-people-selector>
+                    <people-widget [(peopleId)]="assigneeId" [field]="field"></people-widget>
```
Comment (mauriziovitale) [inline lib/process-services/task-list/components/start-task.component.html]: is the people widget able to show the person?
I think we are only using the output and not the two way binding
If so we can change this code

 ```@Output()
+    peopleIdChange: EventEmitter<number>;```
to
 ```@Output()
+    peopleSelected: EventEmitter<number>;```

File: lib/process-services/task-list/components/start-task.component.ts [hunk 2]
```diff
@@ -81,6 +85,7 @@ export class StartTaskComponent implements OnInit {
     }
 
     ngOnInit() {
+        this.field = new FormFieldModel(new FormModel(), {id: 'fakeId', value: 'fakeValue', placeholder: 'Assignee'});
```
Comment (mauriziovitale) [inline lib/process-services/task-list/components/start-task.component.ts]: would be better remove the fakeId and use assigneeId 
same for value you can use assigneeId I guess

Comment (mauriziovitale) [inline lib/process-services/task-list/components/start-task.component.ts]: I don't see a unit test about that. can you add one?

Comment (mauriziovitale) [reply lib/process-services/task-list/components/start-task.component.ts]: Even the documentation is missing

Comment (VitoAlbano) [reply lib/process-services/task-list/components/start-task.component.ts]: I think this highlights a problem of trying to use this widget here . If this is needed like that wouldn't it be better try to create a common one that could be extended by the widget(which has the form) and the one you need?

**Comment by mauriziovitale** [comment]
opened a different issue

**Pull request closed without merging**